slides/week0/week0.org

```diff
@@ -0,0 +1,484 @@
+#+TITLE: Intro to Programming
+#+AUTHOR: Jason Gibson
+#+EMAIL: jgibson37@gateche.edu
+
+* What are we doing today?
+- Instructor introductions
+- What is programming
+- Hello World!
+- Variables
+- Control Flow
+- Conditionals
+
+* Meet Jason
+#+ATTR_HTML: :width 15%
+[[file:https://i.imgur.com/izC5WWA.jpg]]
+- Jason Gibson
+  - Junior, Computer Science (Threads: Devices, Intelligence)
+  - Inside RoboJackets: Training Chair, IGVC Software Lead
+  - Outside RoboJackets: Avid lover of dad jokes
+  - Master of cringe worthy pictures
+- How to contact me
+  - Slack: [[https://robojackets.slack.com/messages/@jasongibson/][@jasongibson]]
+  - Email: [[mailto:jgibson37@gatech.edu][jgibson37@gatech.edu]]
+
+* What Programming Is
+#+BEGIN_NOTES
+Talk for a minute or two about everything programming can do
+#+END_NOTES
+
+* What Programming Is Not
+#+BEGIN_NOTES
+This is meant to emphasize that they will not become a God programmer overnight
+and to dispell the misconception that they are going to be coding amazing things
+immediately.
+#+END_NOTES
+- Impossible
+- Easy
+
+* Programming Failures
+#+BEGIN_NOTES
+Talk for a minute or two about everything programming can do
+Orbiter was pounds instead of newtons
+Robocup forgot the flip the direction on the field and tried (unsuccessfully) to score on themselves
+Jason tried to delete a local directory usr and did sudo rm -rf /usr on a production server
+Ariane software testing did not acccount for increased speed and tried to store a number
+    that was not 16 bits in 1 bits and ended up in an explosion
+#+END_NOTES
+- Mars Climate Orbiter (1998)
+    - the code returned the wrong units causing a 655 million dollar explosion on Mars
+- RoboCup field disaster
+- deleting the /bin directory on a production server
+- Ariane s5 flight 501
+    - faster speed resulted in a 64 bit number being stored as 16 bits ~500 million failure
+
+* Codepad
```

Comment:
I was planning on cropping the middle of the picture out so that it is much larger.